examples/browser/package.json

```diff
@@ -38,6 +38,7 @@
     "coverage": "yarn coverage:compile && yarn test && yarn coverage:remap && yarn coverage:report:lcov && yarn coverage:report:html"
   },
   "devDependencies": {
-    "@theia/cli": "0.2.0"
+    "@theia/cli": "0.2.0",
+    "wdio": "^1.0.3"
```

Comment:
You don't need it. You can call tests from the root or call yarn to install binaries: https://github.com/theia-ide/theia/issues/577#issuecomment-336068209